### PR TITLE
3.0-ify start/first-element/*

### LINKED ISSUE #2538: 3.0-ify start/first-element tutorial

### PR DESCRIPTION
**Pull request by katejeffreys**
Staged here: https://3-0-first-element-dot-polymer-project.appspot.com/3.0/start/first-element/intro

Uses https://github.com/PolymerLabs/polymer-3-first-element

fixes #2538

### CODE REVIEW

File: app/3.0/start/first-element/intro.md [hunk 1]
```diff
@@ -5,87 +5,77 @@ subtitle: "Build your first Polymer element"
...
+### Install dependencies and run the demo {#run}
 
 To install the element's dependencies and run the demo:
 
 1.  Run `yarn install` from the repo directory:
 
-        yarn install
-
+    ```
+    yarn install
```
Comment (keanulee) [inline app/3.0/start/first-element/intro.md]: We don't use yarn anymore - use `npm install` and package-lock.json instead

**Comment by katejeffreys** [comment]
@arthurevans ready for you to look at

**Comment by katejeffreys** [comment]
- [ ] arthur review

**Comment by arthurevans** [comment]
I noticed one style thing I think we should do differently, but nothing that should block merging. I opened that as https://github.com/Polymer/docs/issues/2552.

**Pull request merged**